### DISCUSSION TITLE
Switch to districts

### DISCUSSION THREAD

**Discussion by CAWales** [code4recovery/12-step-meeting-list#612] [Q&A] [answered]
Hi

We currently only have the option on our website to filter meetings by region - https://meetings.cawales.org.uk/meetings

Another area has the option to switch to districts in their filter option - https://meetings.cocaineanonymous.org.uk. How can we enable this function please? Each group has been assigned a district, we just don't currently have the option to filter through this option. 

If coding is required, please give clear cut directions on where this needs to go as we're novices :)

Many thanks in advance for your help and for creating this amazing, user friendly plug in :)

**Comment by Grateful-Viv** [comment]
Have you entered any districts for your meetings?

**Comment by brianw-area24** [comment]
As @Grateful-Viv suggested, I looked at a few meetings, and I'm not seeing any district information on the meetings I looked at. In order to input District information, the meeting needs to be entered as "Part of a group", not an individual meeting, and I think you'd need at least one group for each District you want to enter.

**Comment by CAWales** [comment, marked as answer]
Hi, I have entered district information for every meeting. Unsure why it’s not showing?

**Reply by CAWales** [reply]
All fixed. Did not enter the 'group' field for each meeting, therefore the district information did not save. Have gone in and updated all meetings and now the option to switch to districts is available. Thank you :)
